src/RightsService.Data/UserRepository.cs

```diff
@@ -88,6 +88,14 @@ public async Task<DbUser> GetAsync(Guid userId)
       return await _provider.Users.Include(x => x.Rights).FirstOrDefaultAsync(x => x.UserId == userId);
     }
 
+    public async Task<List<DbUser>> GetUsersAsync(List<Guid> rolesIds)
```

Comment:
you do not need use add the word "user" to method name in UserRepository

Reply:
fixed